### PR TITLE
Cleanup utility projects

### PR DESCRIPTION
**Pull request by lahm86**
I'll group projects together into separate cleanup PRs to try to keep the file numbers a _bit_ more manageable.

This one includes the 5 utility projects - each cleanup action is in its own commit.

Part of #425.

### CODE REVIEW

File: TRLevelToolset/IOLogic/IOManager.cs [hunk 1]
```diff
@@ -1,136 +1,129 @@
...
+    public static TRLevelBase CurrentLevel { get; set; }
 
-        public static string? FileName { get; set; }
+    public static string FileName { get; set; }
 
-        public static TRGame LoadedGame { get; set; }
+    public static TRGame LoadedGame { get; set; }
 
-        public static TR1Level? CurrentLevelAsTR1 
+    public static TR1Level CurrentLevelAsTR1
```
Comment (rr-) [inline TRLevelToolset/IOLogic/IOManager.cs:14]: Wait, why are we removing the `?`? This function can still return `null`.

Comment (lahm86) [reply TRLevelToolset/IOLogic/IOManager.cs:14]: It's currently only called in the likes of the situation below, so we still check for null at that end. I find this approach cleaner, the VS warnings when `nullable` is on are less than helpful I find.

https://github.com/LostArtefacts/TR-Rando/blob/master/TRLevelToolset/Controls/DataControls/TR/TRAnimatedTextureControl.cs#L19

This is the UI when the level is null.

image

Comment (rr-) [reply TRLevelToolset/IOLogic/IOManager.cs:14]: I personally think it's cleaner to leave these intact. The warnings are not wrong, even if they're annoying.

Comment (lahm86) [reply TRLevelToolset/IOLogic/IOManager.cs:14]: That's fair. I'll re-enable and look at clearing the warnings.

**Review by rr- — APPROVED** [review]

**Pull request merged**